torch/ao/quantization/utils.py

```diff
@@ -360,3 +361,26 @@ def _parent_name(target):
         return '', r[0]
     else:
         return r[0], r[1]
+
+def nonparam_type(module):
```

Comment:
can we call it something like `type_before_parametrizations`?

Reply:
sure